### PR TITLE
Minor punctuation/grammar/wording fixes

### PR DESCRIPTION
**Pull request by coreyhaines**
Had some missing punctuation marks! Let's continue to make the README for Cornelius the best that it can be!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# I would like to introduce you to my [Battlesnake](http://play.battlesnake.com?utm_source=github&utm_medium=readme&utm_campaign=go_starter&utm_content=homepage) friend Cornelius
+# I would like to introduce you to my [Battlesnake](http://play.battlesnake.com?utm_source=github&utm_medium=readme&utm_campaign=go_starter&utm_content=homepage) friend, Cornelius
 
 ![a corn snake](https://user-images.githubusercontent.com/8124558/132770698-160c6a6a-5b30-4b7d-9448-8982010b2027.jpg)
 
-Cornelius is built with Rust and is a wiggly darling.
+Cornelius is built with Rust, and he is a wiggly darling.
```
Comment (ChaelCodes) [inline README.md:5]:
```suggestion
Cornelius is built with Rust, and they are a wiggly darling.
```
Cornelius uses they/them. I have trouble with their pronouns too! Let's practice with Cornelius.

**Review by ChaelCodes — CHANGES_REQUESTED** [review]
I really appreciate this PR! It makes me so happy to see the love and care for Cornelius's README.

File: README.md (hunk 1 above)
Comment (coreyhaines) [reply README.md]: Thanks!

**Review by ChaelCodes — APPROVED** [review]
Thanks for this! I've always had issues with comma splices.

**Pull request merged**